user_guide_src/source/changelogs/v4.6.1.rst

```diff
@@ -30,6 +30,8 @@ Deprecations
 Bugs Fixed
 **********
 
+- **CURLRequest:** Fixed returning multiple header sections as body after redirects.
```

Comment:
Can you be clearer as to what changed here?

Reply:
"Fixed an issue where header section(s) are returned in CURL response body, if `allow_redirects` is set and there are redirects from target server."

Is this ok with you?

Reply:
How about this: "Fixed an issue where multiple header sections appeared in the CURL response body during multiple redirects from the target server."

Reply:
I've pushed a change to that :+1: